dev/TeachingTip/TeachingTip.xaml

```diff
@@ -113,7 +113,6 @@
                                 <VisualState x:Name="NoIcon">
                                     <VisualState.Setters>
                                         <Setter Target="IconBorder.Margin" Value="{StaticResource TeachingTipIconPresenterMarginWithoutIcon}"/>
-                                        <Setter Target="IconBorder.Child" Value=""/>
```

Comment:
Did you try {x:Null}

Reply:
Yeah, when a property is set from code behind vsm setters stop working on that property. Since we need to do the conversion from IconSource to Icon in code behind and we set the IconBorder.Child to that result we can't actually do the NoIcon case from VSM as I was hoping.

Reply:
Should we do it as a TemplateSettings calculated value? I still think setting it from VSM makes sense.

Reply:
Hmm, yeah, that might be best.

Reply:
It ended up being a binding on the child property of the border instead of using the VSM, I think this is good though.